### DISCUSSION TITLE
Grove - CO2 Sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1249] [Q&A]
# Grove - CO2 Sensor | Seeed Studio Wiki

Grove - CO2 Sensor

https://wiki.seeedstudio.com/Grove-CO2_Sensor/

**Comment by rastengineering** [comment]
Can I run this sensor and the sample code with a WIO Terminal, powered by Laptop USB?
